test/test_selector_test.rb

```diff
@@ -7,7 +7,16 @@ def setup
       target_sha = @repo.branches['change_fizz'].target.oid
       master_sha = @repo.branches['master'].target.oid
       base_sha = @repo.merge_base(target_sha, master_sha)
-      @selector = TTNT::TestSelector.new(@repo, target_sha, base_sha)
+
+      # NOTE: Use the same configuration for Rake::TestTask with fixtures/Rakefile
+      #       to create TTNT::TestTask which are consistent with actual data
+      rake_task = Rake::TestTask.new(:test) do |t|
+        t.libs << "test"
```

Comment:
Do you really think that we should maintain a similar Rake task that the one in the fixture Rakefile ? I think we should only keep what's necessary for testing purpose ; there are edge cases anyway, the fixture file is just a general one.